kpatch-build/Makefile

```diff
@@ -10,19 +10,18 @@ SOURCES = create-diff-object.c kpatch-elf.c \
 		  create-kpatch-module.c \
 		  create-kpatch-module.c lookup.c
 
-ifeq ($(ARCH),x86_64)
 SOURCES += insn/insn.c insn/inat.c
 INSN     = insn/insn.o insn/inat.o
 insn/%.o: CFLAGS := $(filter-out -Wconversion, $(CFLAGS))
-else ifeq ($(ARCH),ppc64le)
+ifeq ($(ARCH),ppc64le)
```

Comment:
It's probably not a big deal, but believe it or not, there are people out there who cross-compile for x86 on a ppc64le host.  Does the plugin not compile on x86?

Reply:
I haven't looked at the plugin just yet.  The scope of this PR was to only incrementally help create-diff-object so that it may read/write cross-arch ELF for the pre-built unit test objects.  AFAIK, the plugin only comes into play for the kernel builds, right?

Reply:
Ok, makes sense